sources/US/WA/wildlife-areas-2017.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name" : "WA Wildlife Areas",
+    "url": "http://geo.wa.gov/datasets/89e677d17858438b9befb2963f0bd92f_0.geojson",
+    "species": [
+       "all"
+    ],
+    "attribution": "Washington Department of Fish and Wildlife",
+    "properties": {
+       "id": "GMU_Num",
```

Comment:
It doesnt look like the properties in the file match the properties listed here. 
<img width="1064" alt="screen shot 2017-08-28 at 9 29 58 am" src="https://user-images.githubusercontent.com/2048291/29780630-b6c6bf20-8bd3-11e7-9b06-9a2bf02a9078.png">